Python Serverless AI documentation (#898)
fermyon/developer: content/spin/serverless-ai-api-guide.md

{{ startTab "Python"}}

```python
from spin_http import Response
from spin_llm import llm_infer

def handle_request(request):
    prompt="You are a stand up comedy writer. Tell me a joke."
    result=llm_infer("llama2-chat", prompt)
    return Response(200,
                    {"content-type": "text/plain"},
                    bytes(result.text, "utf-8"))
```

**General Notes**

`llm_infer` operation:

- It takes in the following arguments - model name and `prompt`. 

> - The model name is just passed in as a string for now (as you can see from above; `"llama2-chat"`). Future work may use inbuilt model definitions. For example, a Python [dataclass](https://docs.python.org/3/library/dataclasses.html) approach with a pre-defined `InferencingModelLlama2Chat` variable being passed in as the model (instead of the `"llama2-chat"` string) and a variable called `InferencingModelCodellamaInstruct` (instead of a`"codellama-instruct"` string). For now, we just use the string approach.

Reviewer: Personally, I don't think this item needs to be here.  It's true that we might change the API to use a different data type to represent the model in the future, but I don't think what we're currently doing is wrong or would lead to confusion such that it needs a special explanation.

> - The model name is just passed in as a string for now (as you can see from above; `"llama2-chat"`). Future work may use inbuilt model definitions. For example, a Python [dataclass](https://docs.python.org/3/library/dataclasses.html) approach with a pre-defined `InferencingModelLlama2Chat` variable being passed in as the model (instead of the `"llama2-chat"` string) and a variable called `InferencingModelCodellamaInstruct` (instead of a`"codellama-instruct"` string). For now, we just use the string approach.

Author: ```suggestion
- The model name is passed in as a string (as shown above; `"llama2-chat"`).
```
